### DISCUSSION TITLE
Forks view should filter/sort

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#45073] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

GitHub Mobile beta

1. Open https://github.com/facebook/react
2. Tap 41.7k forks

### Expected results

Be able to filter/search for forks by name of owner/fork, to sort based on age of commits to the default branch/number of forks/stars/issues/pull requests

### Actual results
![Screenshot_20230122-112609](https://user-images.githubusercontent.com/2119212/213927088-8de5238f-140c-482e-9e9b-a7ea7c87025b.png)

Scroll down
![Screenshot_20230122-112627](https://user-images.githubusercontent.com/2119212/213927103-aaad2ed3-5afe-4fc3-9d92-7d59fc222035.png)

For reference, the web version shows a list sorted by owner name:
![Screenshot_20230122-112841](https://user-images.githubusercontent.com/2119212/213927168-d5a7aa38-308a-4e84-982e-54f7ec5bbea4.png)

**Comment by iskae** [comment, marked as answer]
Hi @jsoref 👋 

Thanks so much for reaching out 🙇 I believe we are currently not offering a search on forked repositories list based on user name in web too, but this is definitely an area for improvement 👍 

Created an internal tracking issue and we will be discussing further how we can handle this 🎉 

Please keep the feedback/suggestions coming ❤️

**Reply by paiwand61** [reply]
![image](https://github.com/community/community/assets/170662124/bcd77875-5a45-4e11-83f3-001d70079a73)

**Comment by ourslfs** [comment]
+

**Comment by suryanarayan-nadar** [comment]
Dsno

**Comment by crackwitz** [comment]
Web or app, the list of forks is... not all that useful to me.

A whole lot of people "fork" popular repos for no other reason than to "hang it on their wall". No commits on top, no PRs to upstream. Just a fork to decorate an otherwise empty account.

I want all those pointless forks to be deranked. I want to see forks with activity and stars first. Not *recent forks*. Recent *activity* by amount, not time.

That would help me see if there's a community revival of some otherwise dead/unmaintained project.
